manifest.json

```diff
@@ -1,11 +1,14 @@
 {
-  "name": "ts-template",
-  "description": "ts-template for UbiquityOS plugins.",
-  "ubiquity:listeners": ["issue_comment.created", "issue_comment.deleted", "issue_comment.edited"],
+  "name": "scan-contributions",
+  "description": "UbiquityOS plugin for scanning contributions based on issues/pulls.",
+  "ubiquity:listeners": [
+    "issue_comment.created",
+    "issue_comment.edited"
+  ],
   "commands": {
     "command1": {
-      "ubiquity:example": "/command1 argument",
-      "description": "Command 1 with an argument."
+      "ubiquity:example": "/scan-contributions argument",
+      "description": "Activating command with an argument."
```

Comment:
Probably could use a better description.

Reply:
> Probably could use a better description.

Would work on it.